README.md

```diff
@@ -58,26 +58,22 @@ Check out this screencast on how to create and deploy a new Shopify App to Herok
 
 Or if you prefer text instructions the steps in the video are written out [here](https://github.com/Shopify/shopify_app/blob/master/docs/Quickstart.md)
 
-Becoming a Shopify App Developer
---------------------------------
-If you don't have a Shopify Partner account yet head over to http://shopify.com/partners to create one, you'll need it before you can start developing apps.
-
-Once you have a Partner account create a new application to get an Api key and other Api credentials. To create a development application set the Application Callback URL to
+App Tunneling
+-------------
 
-```
-http://localhost:3000/
-```
+In order to install your local app on a shop, use the [App Proxy Controller](#app-proxy-controller-generator) or receive Webhooks from Shopify, your app needs to be on a publicly visible URL. Use a tunneling service like [Forward](https://forwardhq.com/), [RequestBin](https://requestb.in/), [ngrok](https://ngrok.com/) etc. These tools allow you to create a secure tunnel from the public Internet to your local machine.
```

Comment:
First sentence is a [garden-path sentence](https://en.wikipedia.org/wiki/Garden-path_sentence) for me — I interpreted it as “In order to install your local app on a shop, [you should] use the App Proxy Controller…”.

Maybe reword as “Your local app needs to be on a publicly visible URL in order to install it on a shop, use the App Proxy Controller, or receive Webhooks.”

Reply:
Ah yes, updated.

Reply:
Yes, requestb.in does not exist anymore.

You can recommend one of these as an alternative:
https://beeceptor.com/
http://mockbin.org/
https://hookbin.com/

Reply:
good catch, i'll do a new PR to fix that

Reply:
i pushed it to master instead of doing a PR